src/scaffold.ts

```diff
@@ -220,7 +220,14 @@ async function pathExists(path: string) {
 }
 
 async function pickTemplate(templateList: Template[]): Promise<vscode.QuickPickItem | undefined> {
-  const quickPickItems: vscode.QuickPickItem[] = templateList.map((templateItem: Template) => {
+  const sortedList = templateList.sort((a, b) => {
+    const nameA = a.spec.display_name.toLowerCase();
+    const nameB = b.spec.display_name.toLowerCase();
+    if (nameA < nameB) return -1;
+    if (nameA > nameB) return 1;
+    return 0;
```

Comment:
Also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
thanks for the suggestion ✔️